include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -808,6 +808,34 @@ FUNCTION(UpdateClassMetadata,
               SizeTy->getPointerTo()),
          ATTRS(NoUnwind))
 
+// struct FieldInfo { size_t Size; size_t AlignMask; };
```

Comment:
Just a comment fix, but what is this "FieldInfo"? Is it supposed to be the "TypeLayout" type mentioned in the comment's signature?

Reply:
Yes.  I just copied that comment from another declaration, but I can fix it.